docs/data/toolpad/connecting-to-datasources/google-sheets.md

```diff
@@ -0,0 +1,29 @@
+# Google Sheets datasource
+
+<p class="description">Google Sheets datasource allows fetching data from Google Sheets living in your Google Drive.</p>
+
+## Working with Google Sheets
+
+As explained in the [connections](/toolpad/connecting-to-datasources/connections/) section you can either create a reusable connection or simply create a new query and put all connection details inline:
```

Comment:
I don't understand how the "or put all connection details inline" would work here, what is that supposed to mean?

Reply:
Thanks, that's a good catch 👍 I did copy/paste this from function/fetch...